InventoryInStorePickupAdminUi/Test/Mftf/ActionGroup/AdminAssertOrderStatusActionGroup.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AdminAssertOrderStatusActionGroup">
+        <annotations>
+            <description>Verify order status on admin order page.</description>
+        </annotations>
+        <arguments>
+            <argument name="orderStatus" type="string" defaultValue="{{OrderStatus.processing}}"/>
+        </arguments>
+        <see selector="{{AdminOrderDetailsInformationSection.orderStatus}}" userInput="{{orderStatus}}" stepKey="seeOrderStatusCanceled"/>
```

Comment:
Is this action group supposed to work with other types of statuses? Either group name or step key is incorrect